lib/fluent/plugin/out_graphite.rb

```diff
@@ -49,19 +49,26 @@ def configure(conf)
     end
     # How many times to retry the call if timeout raised
     @max_retries ||= 3
+
+    if @flush_interval < 10
+      log.info("flush_interval less than 10s is not allowed and overwritten to 10s")
+      @flush_interval = 10
```

Comment:
> Ah, no, I meant that, "isn't it a problem to forcibly set it to 10 seconds even if the setting on the graphite side is not 10 seconds?"

Strictly speaking, there is *still* problem to time resolution.
But current implementation contains more problem.
With current implementation, Fluentd events cannot wait even if 1 second.
Non-Buffered plugin pass through events with no wait time.